docs/core/compatibility/categories.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Breaking change categories
+description: Learn about the ways in which breaking changes are categorized.
+author: rpetrusha
+ms.author: ronpet
+ms.date: 06/10/2019
+---
+# Breaking change categories
+
+*Compatibility* refers to the ability to compile or execute code on a version of a .NET implementation other than the one with which the code was originally developed. Changes between versions can affect compatibility in A particular change can affect compatibility in six different ways. The [individual kinds of changes that are considered when evaluating compatibility](index.md) fall into the first five categories. These are:
```

Comment:
can affect compatibility in A particular change -> it seems you forgot part of the sentence here?